_sass/_base.scss

```diff
@@ -19,3 +19,31 @@ svg {
Af1W8PcDdwEfg78AYl0QZlQ0XEbSuB+SwxYYbl20tqPwQcpywEFfB1YL/wu114/AH4GnBefiwQcRuSnffL9gvABPA28BdKgtXlCpGA3yfy4jUC+1FiApwlYtYn/Q8CZ4DvEHk3CxxS+3H1McZLum7I5y3AJWExLds9sXcJOy+stg3KwvqgbFggYrtLYw4Loy1qf4nykkDPDwA7dT1AxP+8no0RuZQx8ssFygt5hMhpiNgcJfIS4Bki/kek6wMiJgeJPBjR5zARy5Z0niXy4KL+PhI2o/KhKbvuE94DBP6vC4cxYs405d/t8uuoxviU1S+Tvg6fW4ncHpYNEwPAz4ggnyES43E5P0UsHN7BvAVMEon3KvAAEZRBYtKNaPB+4Hu6P04kyTzwDfW5G/iRDDijvuNyuEEsPvuAr1J2WVeIRJvVvTsVCCiLU5NIqleAGQHbls2LstuTHCLRZgRsv+zYn0B8TH4gm5rJhnkiUE318cI0SllgnKgTRPLMyZaPhe35hJntHAMeFXa75M+AsNipsYeF5TQl0X6iT+PQErYLwEO6/3ONPQq8QCyetuk5jTVFvJCGiThV6vMQMVkP6NlxYpJOJh1zwJOUhSG/RceJJL9bPnqR/pbG+CHwsvSitgeF3T0af5qI2U4i/uf03G9kL7jvSO/ZhMcikTeniXidkC2npW8+xWK3xvxQtj0JbCcWou1E/kyoreN9Qd8n5cscEd9pYhGfTe0XNcZMsulOPZ8k8m63rv0SPUjJgUEiRr8g5uS0+rV0/wTwbPIL4LJseCrpnQcOEzE/QczNaSJWO4k4P6y/Sj4doOxIjVWTeLl9RonxaSK/7NtFYsGqNN5eIr/337bv+UMXNNg8MWG8hfR211tjb+tHdK+ibDkdPIu3jv6+RAlypXE8aatavyU5Nd7hWT5W1I9nsPqY0a/vPu4Mc/W218fOirL1b6dnFauPHhbvgKBs6S0+Srmfk78luy+pTz7y5COb9Q2qrXWO6H7eNhuH2ykTvh436/UiXhGxyOMYL8uQ2tju0dTXukl9lzX+POVYZN8sS5S4D6f7OSYNSu55l+Hccmxtp2Obj+s+MrmPMbB/blv/bLH6WOc8qIgYuG09blDwdb4NcfWbv0WJhedOzgtPauNZ93eFq+eY2/iFaTs8d3yvfgQzvj7+2rfsk+OZMXNbHxfzkc65ldsOsRrn+s8FbaDvtn3PH/pPB8duyS25Jbek51L/MbQX0mv93cqN2LORvp36tNd5/r8o1/Kz/qy9xv3NGq+bNmv16bbvlyHOnXzrqZ99rP2j4YaV1uR69Pdasv0bsafTcfFGxstb45sBH+guxhtJyk5+1jHI12v1uR6b6j8BXGvcbn2pap/Xa9NmxbmXL69Ovl0Lx2vpuC7pW79J90pvQrlR+7vtfz3tbzZMu7Fns/Co3++U+BsZYz09NzJON9IL3Z+X7dcad9PkehegL5N8UVvdGz2e9XIH0q2eXh8vvgzHkS9a/i8w6mYB2sgEWe/3j82Qur7NWKm7XRi6PS5Y1nubrWXHRo4mnX5nWWvMnm67N9C+F1LHZTNzdTN+K7yZMOpFnzZcvQBdq/NGtujrbX83Q26GbW6vkmUzttv1WKz329MXtcX/vKWOy2bm6o2+jG4W6aUfFRTuhrkJZhqbel/nSuT/7ZsTYjG/wHyO3NY66xT6zEGoWM0vcNv8Q+E8hUPie0OpbZYGhUNR1dqYt1PnYmTeUD9rczrMgcp22CfjYN/MfTEHpF76sBY3JfNXjHNDdpqj5LaOHx38dXlK5h+Zz5FLMkh63dZS5/aMszqeuUwn840yvmYvW3KphDlJ46lvLnMxBi65MH6XEw4mrTqPGhS+0pbaONZr/3PctiR9maNT53dR05cZzVnqPLScB86dtThqmTeUeUed8t04oOd1Dp3nzmCHZ+5/icLnGa+17bRjblDik3lZHr8eN3ObLkl/c0AXxwjm7GkK9f4ywfj8hMJq3EtQuLcRrNOn5dRMMmqRoG9PSs8u3W9SWMR7KOzL8wTTc5xgmrqt6d7o2Szwa/V5m0iQOel6VG0eIhJiFniXwjC13hngTfUj+daisFNzvctOAbmHYHBOAi+q/yKFru/2FwkW+TmCcn5OY7cp7Fpj5SDPJlwfIGJhm0YpC+4OonzhYYJ1arr/qMbdLn9mCWb0HuB9jXmRYL2aBf4gwfJ+UbbNESzZt+SPY9yJ0TwE/JSrWdDHpbdJxH9A49+lMZ7Ts7cJVvodwO+TjtMEW3tRffZq3Pcp5TtDrGZ8/0mfp+TvNoLJb9b4u7L720R+LyUf5ygTqkFhp9dZ6weJObJfbXO8KyJeLncxi7jSPZervEzEH/lwTP6syO7dlNIcl/ycJGJrO82gfopSYgNlQa6I+LeSDV40W8Ix2+Q6tXdkywRRSvOR8H5W/riawS8IL4QN9TXJ+ARRUXFSNj9KeWnNAD8gykaO6PlzwAsDwK9QXQYRzPeIYK1Vz2EQtwK/IWpVpih072Fikrt+5IAcXNbg71OKGLcJ2MMEhXtCf/cRE85v0MsUSv1hItiH5eC9suVxoh7tb7LxY0odDXK+IR259iazctvStaDxpgTca3o+S0wuM6jtQ702yJPwDLEQma5/nEi0emGvi09H1cdvwTEiQT+l1Df9jtjBDFHeUF4Y5il1TjuF96vC/7hsuEIsVjvUZ1E4TQL/JGrNXIDYSdpq9wiRN17UTkq/y0BcZzYlLD6SvZeJiX6MiE2bUph6Wv0XiZeby13szw7Z9pLanxKWrot7Qu0nKLVOy8J2msjtP1MmeWZq36HPhXTfi3tFTLZJfebav1OyO++AKuHiOsIWMQ8WiLys5Nug7r2pP4+PMDqr6xEiZhVR67hftn2QbH9A7euseoj8cn3lEUrd5SXgH2pzJ6V85x4ivjME5q9I535hvULUo50n8vNf6vuG+h+RTYOyc7vaviTsvCYc+i9pQJWwuXILrwAAAABJRU5ErkJggg==);
```

Comment:
Does SASS have a way to inline images like this? I feel like it'd be more maintainable if you had the images in external files, and used a SASS function to inline them.